modules/mobile/docs/openapi.yaml

```diff
@@ -131,68 +131,6 @@ paths:
       security:
         - Bearer: []
       summary: /v0/appeal/{id}
-  /v0/appointment_requests/{appointment_request_id}/messages:
```

Comment:
Should this be "deprecated" rather then deleted? or noted in some way that this doesn't exist anymore rather then removing all references to it.

Reply:
I'd vote for just deleting it. These docs are already a pain to keep up to date. If for some reason we ever need this, git has it for us.